### PR TITLE
deployment-arbitrary-cluster.md: Fix instructions.

### PR DESCRIPTION
**Pull request by russellb**
The sample commands quoted the list of filenames, which makes them
a single item.  The whole list of filenames is then used in a single kubectl
command.  Just drop the quotes so each filename is handled by itself.

Also change kubectl to oc for the OpenShift instructions.

Create the nmstate namespace before applying manifests.

Fix paths to CRD and openshift SCC manifest.

Add a missing CRD to the instructions.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by qinqon** [comment]
Thanks !

**Comment by russellb** [comment]
you're welcome - with these changes it works on my cluster.  :+1:

File: docs/deployment-arbitrary-cluster.md [hunk 1]
```diff
@@ -41,18 +41,24 @@ Finally, we can install kubernetes-nmstate on our cluster.
 ### Kubernetes
 
 ```shell
+# Create the “nmstate” namespace
+echo '{"apiVersion":"v1","kind":"Namespace","metadata":{"name":"nmstate","labels":{"name":"nmstate"}}}' | kubectl apply -f -
```
Comment (qinqon) [inline docs/deployment-arbitrary-cluster.md]: We have a namespaces.yaml already let's use it

File: docs/deployment-arbitrary-cluster.md [hunk 2]
```diff
@@ -41,18 +41,24 @@ Finally, we can install kubernetes-nmstate on our cluster.
...
+
 # Install k8s resources and kubernets-nmstate operator
-for manifest in "service_account.yaml role.yaml role_binding.yaml crds/nmstate_v1alpha1_nodenetworkstate_crd.yaml operator.yaml"; do
+for manifest in service_account.yaml role.yaml role_binding.yaml nmstate_v1alpha1_nodenetworkstate_crd.yaml nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml operator.yaml; do
```
Comment (qinqon) [inline docs/deployment-arbitrary-cluster.md]: put namespaces.yaml as the first one

File: docs/deployment-arbitrary-cluster.md [hunk 3]
```diff
@@ -41,18 +41,24 @@ Finally, we can install kubernetes-nmstate on our cluster.
...
     kubectl apply -f https://github.com/nmstate/kubernetes-nmstate/releases/latest/download/$manifest
 done
 ```
 
 ### OpenShift
 
 ```shell
+# Create the “nmstate” namespace
+echo '{"apiVersion":"v1","kind":"Namespace","metadata":{"name":"nmstate","labels":{"name":"nmstate"}}}' | oc apply -f -
```
Comment (qinqon) [inline docs/deployment-arbitrary-cluster.md]: ditto

File: docs/deployment-arbitrary-cluster.md [hunk 4]
```diff
@@ -41,18 +41,24 @@ Finally, we can install kubernetes-nmstate on our cluster.
...
+
 # Install k8s resources , openshift scc and kubernets-nmstate operator
-for manifest in "service_account.yaml role.yaml role_binding.yaml openshift/scc.yaml crds/nmstate_v1alpha1_nodenetworkstate_crd.yaml operator.yaml"; do
-    kubectl apply -f https://github.com/nmstate/kubernetes-nmstate/releases/latest/download/$manifest
+for manifest in service_account.yaml role.yaml role_binding.yaml scc.yaml nmstate_v1alpha1_nodenetworkstate_crd.yaml nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml operator.yaml; do
```
Comment (qinqon) [inline docs/deployment-arbitrary-cluster.md]: ditto

**Comment by qinqon** [comment]
@russellb have re-review it, I forgot about namespace.yaml

**Comment by qinqon** [comment]
Don't know why standard-ci is stuck, will take a look.

**Comment by qinqon** [comment]
ci test please

**Pull request merged**